frontend/src/scenes/onboarding/OnboardingProductIntroduction.tsx

```diff
@@ -0,0 +1,250 @@
+                        <aside className="text-right my-2 hidden md:block">
+                            <img src={product.image_url || undefined} className="max-w-96" />
+                        </aside>
+                    </div>
+                </header>
+                {product.screenshot_url && (
+                    <div className="flex justify-center">
+                        <div className="max-w-6xl mt-8 -mb-12">
+                            <img src={product.screenshot_url || undefined} className="w-full" />
+                        </div>
+                    </div>
+                )}
+
+                <div className="features p-8 py-8 border-t border-border flex justify-center">
+                    <div className="max-w-screen-xl">
+                        <h3 className="mb-6 text-2xl font-bold">Features</h3>
+                        <ul className="list-none p-0 grid grid-cols-2 md:grid-cols-3 gap-8 mb-8 ">
+                            {product.features
+                                .filter((feature) => feature.type == 'primary')
+                                .map((feature, i) => {
+                                    return (
+                                        <React.Fragment key={`${product.type}-feature-${i}`}>
+                                            <Feature {...feature} />
+                                        </React.Fragment>
+                                    )
+                                })}
+                        </ul>
+
+                        <ul className="subfeatures list-none p-0 grid grid-cols-2 md:grid-cols-3 gap-4">
+                            {product.features
+                                .filter((feature) => feature.type == 'secondary')
+                                .map((subfeature, i) => {
+                                    return (
+                                        <React.Fragment key={`${product.type}-subfeature-${i}`}>
+                                            <Subfeature {...subfeature} />
+                                        </React.Fragment>
+                                    )
+                                })}
+                        </ul>
+                        <div className="mt-16">
+                            <h3 className="mb-4 text-lg font-bold">Get the most out of {product.name}</h3>
+                            <ul className="flex flex-col sm:flex-row gap-x-8 gap-y-2">
+                                <li>
+                                    <Link to={product.docs_url} target="_blank">
+                                        <IconStack className="mr-2 text-xl" />
+                                        <span className="font-bold">Product docs</span>
+                                    </Link>
+                                </li>
+                                <li>
+                                    <Link
+                                        to={`https://posthog.com/tutorials/${websiteSlug[product.type]}`}
```

Comment:
Feature flags ones goes to :https://posthog.com/docs/feature-flags/installation

Reply:
huh good find, I raised this for website team https://posthog.slack.com/archives/C01V9AT7DK4/p1706220991452339